accounts/urls.py

```diff
@@ -3,6 +3,7 @@
 urlpatterns = [
     path('register/', UserRegistrationView.as_view(), name='register-user'),
     path('login/', UserLoginView.as_view(), name='login-user'),
-    path('verify-email/', VerifyEmailView.as_view(), name='verify-email')
-
+    path('verify-email/', VerifyEmailView.as_view(), name='verify-email'),
+    path('resend-code/', ResendVerificationCodeView.as_view(), name='resend-code')
```

Comment:
can we add api/ in every endpoint or can we reuse it by defining it in the root app